AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/identity/identity-edit/identity-edit.component.ts

```diff
@@ -59,6 +71,16 @@ export class IdentityEditComponent {
         this.getIdentity();
     }
 
+    getAdmissibleTiers() {
+        this.tierService.getTiers().subscribe({
+            next: (tiers) => {
+                this.tiers = tiers.result;
+                this.tier = this.tiers.filter((t) => t.id === this.identity.tierId)[0];
```

Comment:
Is this operation safe? Shouldn't we check the result/length of the collection post filter statement?

Reply:
In that case I would simply use `this.tiers.find((t) => t.id === this.identity.tierId)!`